views/partials/footer.ejs

```diff
@@ -1,42 +1,37 @@
 <footer>
-  <% if (typeof hideFooterCta === 'undefined' || ! hideFooterCta) { %>
```

Comment:
Does this remove it from other pages too? Like this section's also on subpages like /referrals. I think you could just add `hideFooterCta: true` to the `locals` object in https://github.com/shinetext/aurora/blob/dev/api/controllers/WebViewController.js#L137

Reply:
oops, didn't realize other pages use this section. I will revert this change but removing the logic below around the homepage view (lines 5 - 9)